webdriver-spec.html

```diff
@@ -5708,6 +5708,12 @@ <h3>Element Click</h3>
   any <a data-lt="navigation">navigations</a> triggered by the click
   to start.
 
+  <p class="note">It is not always clear how long this will cause the
+  algorithm to wait, and it is acknowledged that some implementations
+  may have unavoidable race conditions. The intention is to allow a
+  new attempt to <a>navigate</a> to begin so that the next step in the
+  algorithm is meaningful.
```

Comment:
I think this misses the point @whimboo raised that not all clicks induce navigation. For example clicking an anchor won’t cause navigation, but the point of this step is to either allow enough time for a navigation attempt to begin in cases you can’t tell up front or for steps to be taken in cases where you can.

So the point here is not that this causes “unavoidable race conditions”, but that there is a need to run some steps to tell whether the click is going to cause or have caused navigation.

Reply:
Surely that's handled by the "any navigations" language? I'll amend note to indicate that not all clicks cause navigation.

Reply:
Thanks, that makes it clearer.

But is it really true that “implementations may have unavoidable race conditions” with running the Go command?